views/editor/publish.html

```diff
@@ -16,10 +16,17 @@
     </div>
 
     <div id="publish-live" class="hide">
+     
       <p>{{ gettext("publishShareLink") }}</p>
       <div id="publish-link">
         <a id="link-publish-link" title="{{ gettext("publishShareLinkTitle") }}" target="_blank" href="test"></a>
       </div>
+      
+        <!--   Embed iFrame   -->
```

Comment:
remove the extra leading spaces before the comment so that it aligns with the tag